src/sdk/mutate_impl.cc

```diff
@@ -258,7 +258,11 @@ void RowMutationImpl::DeleteFamily(const std::string& family,
 void RowMutationImpl::DeleteRow(int64_t timestamp) {
     RowMutation::Mutation& mutation = AddMutation();
     mutation.type = RowMutation::kDeleteRow;
-    mutation.timestamp = timestamp;
+    if (timestamp == -1) {
+        mutation.timestamp = kLatestTimestamp;
+    } else {
+        mutation.timestamp = timestamp;
+    }
 }
```

Comment:
try it on your own risk.

The `-1` is the default value, so we have the duty to check.
But the `-2` is not. 
No zuo no die why you try? You try you high don't ask why. let it go.